Gruntfile.js

```diff
@@ -45,7 +45,7 @@ module.exports = function(grunt) {
                 files: [{
                     expand: true,
                     cwd: 'tmp/',
-                    src: ['*.html', '*.css', '!replace*.*', '!*.php', '!cdnPath.html']
+                    src: ['*.html', '*.css', '!replace*.*', '!*.php', '!cdnPath.html', 'css/*.css']
```

Comment:
No need to add the `css/*.css` glob here as it's in the `rename` task and that is what your tests are covering